runtimes/rust/lbr-prelude/src/json.rs

```diff
@@ -0,0 +1,327 @@
+use data_encoding::BASE64;
+use num_bigint::BigInt;
+use serde::ser;
+use serde_json::{self, Error, Value};
+use std::collections::{HashMap, HashSet};
+use std::hash::Hash;
+
+/// Trait that lbf-prelude::json class maps to
+pub trait Json {
+    fn to_json(&self) -> Result<Value, Error>;
+
+    fn from_json(value: Value) -> Result<Self, Error>
+    where
+        Self: Sized;
+}
+
+//  lbf-prelude::json instance rule implementations
+
+impl Json for BigInt {
+    fn to_json(&self) -> Result<Value, Error> {
+        serde_json::to_value(self)
+    }
+
+    fn from_json(value: Value) -> Result<Self, Error> {
+        serde_json::from_value(value)
+    }
+}
```

Comment:
Can you put a list mapping LB Prelude opaque types into Rust types, and let's have a brief look around that we indeed have chosen what rustaceans would consider proper